es/appendices/orm-migration.rst

```diff
@@ -0,0 +1,15 @@
+ORM Migration Guide
+###################
+
+.. note::
+    La documentación no es compatible actualmente con el idioma español en esta página.
+
+    Por favor, siéntase libre de enviarnos un pull request en
+    `Github <https://github.com/cakephp/docs>`_ o utilizar el botón **Improve this Doc** para proponer directamente los cambios.
+
+    Usted puede hacer referencia a la versión en Inglés en el menú de selección superior
+    para obtener información sobre el tema de esta página.
+
+.. meta::
+    :title lang=es: CakePHP Development Process
+    :keywords lang=es: maintenance branch,community interaction,community feature,necessary feature,stable release,ticket system,advanced feature,power users,feature set,chat irc,leading edge,router,new features,members,attempt,development branches,branch development
```

Comment:
Why does es have an orm migration guide page?

Reply:
I didn't want to remove the inbound link. I removed many `:ref:` links as they are harder to make work, but I felt that adding a 'todo' page was low effort and retained more of the existing translation work.